api/task_processor/health.py

```diff
@@ -0,0 +1,44 @@
+import uuid
+
+import backoff
+from health_check.backends import BaseHealthCheckBackend
+from health_check.exceptions import HealthCheckException
+
+from task_processor.models import HealthCheckModel
+from task_processor.tasks import create_health_check_model
+
+
+def check_processor_health(max_tries: int = 5, factor: float = 0.1):
```

Comment:
Should this be called `is_processor_healthy`?

Reply:
Yes, probably.